indico/modules/users/templates/emails/verify_email.txt

```diff
@@ -5,13 +5,13 @@
 {%- endblock %}
 
 {% block header_recipient -%}
-    {{ user.first_name }}
+    {% trans name=user.first_name %}Dear {{ name }},{% endtrans %}
 {%- endblock %}
 
 {% block body -%}
-{%- trans notrimmed -%}
-You recently added the e-mail address {{ email }} to your Indico account.
-Please click the following link to confirm this action.
+{%- trans notrimmed mail=email -%}
```

Comment:
Why is this needed? Using `{{ email }}` below w/o the assignment here should work as well..